content/en/apps/guides/hosting/app-developer.md

```diff
@@ -0,0 +1,95 @@
+
+
+## Getting started
+
+Be sure to meet the [CHT hosting requirements]({{< relref "apps/guides/hosting/requirements" >}}) first. As well, if any other `medic-os` instances using [the main `docker-compose.yml` file](https://github.com/medic/cht-core/blob/master/docker-compose.yml) are running locally, stop them otherwise port, storage volume and container name conflicts may occur.
+
+After meeting these requirements, download the developer YAML file in the directory you want to store them:
+
+```shell script
+curl -o docker-compose-developer.yml https://raw.githubusercontent.com/medic/cht-core/master/docker-compose-developer-base.yml
+```
+
+To start the first developer CHT instance, run `docker-compose` and specify the file that was just download:
+
+```shell script
+docker-compose -f docker-compose-developer.yml up
+```
+
+This may take some minutes to fully start depending on the speed of the Internet connection and speed of the bare-metal host.  After it has completed, the CHT should be accessible on [https://localhost](https://localhost).
+
+When connecting to a new dev CHT instance for the first time, an error will be shown, "Your connection is not private" (see [screenshot](/apps/tutorials/local-setup/privacy.error.png)). To get past this, click "Advanced" and then click "Proceed to localhost".
+
+## Running the Nth CHT instance
+
+After running the first instance of the CHT, it's easy to run as many more as are needed.  This is achieved by specifying different:
+
+* port for `HTTP` redirects (`CHT_HTTP`)
+* port for `HTTPS` traffic (`CHT_HTTPS`)
+* project to for the docker compose call (`-p PROJECT`)
+
+Assuming you want to start a new project called `the_second` and  start the instance on `HTTP` port `8081` and `HTTPS` port `8443`, this would be the command:
+
+```shell script
+CHT_HTTP=8081 CHT_HTTPS=8443 docker-compose -p the_second -f docker-compose-developer.yml up
+```
+
+The second instance is now accessible at  [https://localhost:8443](https://localhost:8443).
+
+## The `.env` file 
+
+Often times it's convenient to use revision control, like GitHub, to store and publish changes in a CHT app.  A nice compliment to this is store the specifics on how to run the `docker-compose` command for each app. By using a shared `docker-compose` configuration for all developers on the same app, it avoids any port collisions and enables all developers to have a unified configuration.
+
+Using the above `the_second` sample project, we can create another directory to host this project's configuration:
+
+```shell
+mkdir ../the_second
+```
+
+Create a file `../the_second/.env-docker-compose` with this contents:
+
+```shell
+COMPOSE_PROJECT_NAME=the_second
+CHT_HTTP=8081
+CHT_HTTPS=8443
+```
+
+Now it's easy to boot this environment by specifying which `.env` file to use:
+
+```shell
+docker-compose --env-file ../the_second/.env-docker-compose -f docker-compose-developer.yml up
```

Comment:
Need either to reference the `yml` file in the directory above or add a `cp` command to copy it through to the new directory.  I don't think docker will traverse the directories right?

Reply:
Turns out it will! This is literal code I ran to ensure it work.